addons/survey/static/src/js/survey_session_manage.js

```diff
@@ -361,6 +356,8 @@ publicWidget.registry.SurveySessionManage = publicWidget.Widget.extend({
                 });
             } else {
                 self.$('.o_survey_session_close').click();
+                var descriptionDone = self.$('.o_survey_session_description_done').html();
+                self.$('.o_survey_session_manage_container').html("<h1>" + _t('Thank you!') + "</h1>" + descriptionDone);
```

Comment:
When possible we try to avoid injecting html directly. Better rely on simple DOM manipulation and use text to add content when possible in elements, like
```
self.$('.o_survey_session_description_done').prepend($('h1').text(_t('Thank you!')))
```

Reply:
Hi! This was how I initially implemented it, but changed it after lna told me that the description can contain html elements like images (see chatter discussion).